wiki/en/en-Compiling.md

```diff
@@ -62,12 +62,22 @@ Will build the file and make it available in `./Release/Jamulus.app`
 ## Windows
 You don't need to compile Jamulus for Windows since we provide official binaries, but if you really want to, follow the instructions below.
 
-**Required software**: [Qt](https://www.qt.io/download), a compiler like Visual Studio, ASIO development files
+### Required Software
+- A compiler like [Visual Studio](https://visualstudio.microsoft.com)
```

Comment:
Not 100% sure about the MD syntax here. I usually use `*` instead of `-`. But it seems to work this way too. @ignotus666 do you know what the correct way of generating a list is?

Reply:
Either one seems to work but I will change to `*` if that's the convention here.

Reply:
> @ignotus666 do you know what the correct way of generating a list is?

Sorry, I don't really know much about this, but AFAIK in other documents (e.g. software manual) hyphens are used for bullet points, ** for bold and ### for larger font bold.

Reply:
Ok, so probably both are correct.